lib/europeana/blacklight/search_helper.rb

```diff
@@ -19,15 +19,14 @@ def previous_and_next_document_params(index, window = 1)
 
       def fetch_with_hierarchy(id, extra_controller_params = {})
         response, document = fetch(id, extra_controller_params)
-        response.documents.first.hierarchy = europeana_record_hierarchy(id)
+        response.documents.first.hierarchy = fetch_full_hierarchy(id)
         [response, response.documents.first]
       end
 
-      private
-
-      def europeana_record_hierarchy(id)
+      def fetch_full_hierarchy(id)
```

Comment:
Assignment Branch Condition size for fetch_full_hierarchy is too high. [27.09/15]